company_dashboard/src/components/settings/AdminPanel.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+class AdminPanel extends React.Component {
+  state = {
+    name: "company name",
+    motto: "our motto",
+    codeSnippet: "<h1>Welcome</h1>"
+  }
+
+  handleInputChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  render() {
+    return (
+      <div className='create-form'>
```

Comment:
A fine start to the form. Maybe try incorporating Material-UI as you progress so you do less refactoring in the future.